submissions/2020-12-18-OpenFF-Theory-Benchmarking-Set-v1.0/README.md

```diff
@@ -0,0 +1,122 @@
+    4. Using qcsubmit tool, generated 1-D torsions from the selected molecules. 
+
+
+## Manifest
+
+- `QCSubmit workflow.ipynb`: curation of torsiondrive dataset using qcsubmit package.
+- `dataset.json.bz2`: The torsiondrive dataset ready for submission.
+
+## Metadata
+
+```
+{'submitter': 'hyejang',
+ 'creation_date': '2020-12-14',
+ 'collection_type': 'TorsiondriveDataset',
+ 'dataset_name': 'OpenFF Theory Benchmarking Set v1.0',
+ 'short_description': 'Torsiondrives for theory benchmarking',
+ 'long_description_url': 'https://github.com/openforcefield/qca-dataset-submission/tree/master/submissions/2020-12-14-theory-bm-torsiondrive-set-v1.0',
+ 'long_description': 'A torsiondrive dataset for theory benchmarking',
+ 'elements': ['C', 'N', 'Cl', 'F', 'O', 'S', 'P', 'H']}
+```
+
+## QC specification 
+
+```
+{'B3LYP-D3BJ/DZVP ': {'method': 'B3LYP-D3BJ',
+  'basis': 'dzvp',
+  'program': 'psi4',
+  'spec_name': 'B3LYP-D3BJ/DZVP ',
+  'spec_description': 'A torsiondrive dataset for benchmarking B3LYP-D3BJ/DZVP ',
+  'store_wavefunction': 'none',
+  'implicit_solvent': None},
+ 'B3LYP-D3BJ/DEF2-TZVP ': {'method': 'B3LYP-D3BJ',
+  'basis': 'def2-tzvp',
+  'program': 'psi4',
+  'spec_name': 'B3LYP-D3BJ/DEF2-TZVP ',
+  'spec_description': 'A torsiondrive dataset for benchmarking B3LYP-D3BJ/DEF2-TZVP ',
+  'store_wavefunction': 'none',
+  'implicit_solvent': None},
+ 'B3LYP-D3BJ/DEF2-TZVPD ': {'method': 'B3LYP-D3BJ',
+  'basis': 'def2-tzvpd',
+  'program': 'psi4',
+  'spec_name': 'B3LYP-D3BJ/DEF2-TZVPD ',
+  'spec_description': 'A torsiondrive dataset for benchmarking B3LYP-D3BJ/DEF2-TZVPD ',
+  'store_wavefunction': 'none',
+  'implicit_solvent': None},
+ 'B3LYP-D3BJ/DEF2-TZVPP ': {'method': 'B3LYP-D3BJ',
+  'basis': 'def2-tzvpp',
+  'program': 'psi4',
+  'spec_name': 'B3LYP-D3BJ/DEF2-TZVPP ',
+  'spec_description': 'A torsiondrive dataset for benchmarking B3LYP-D3BJ/DEF2-TZVPP ',
+  'store_wavefunction': 'none',
+  'implicit_solvent': None},
+ 'B3LYP-D3BJ/DEF2-TZVPPD ': {'method': 'B3LYP-D3BJ',
+  'basis': 'def2-tzvppd',
+  'program': 'psi4',
+  'spec_name': 'B3LYP-D3BJ/DEF2-TZVPPD ',
+  'spec_description': 'A torsiondrive dataset for benchmarking B3LYP-D3BJ/DEF2-TZVPPD ',
+  'store_wavefunction': 'none',
+  'implicit_solvent': None},
+ 'B3LYP-D3BJ/DEF2-QZVP ': {'method': 'B3LYP-D3BJ',
+  'basis': 'def2-qzvp',
+  'program': 'psi4',
+  'spec_name': 'B3LYP-D3BJ/DEF2-QZVP ',
+  'spec_description': 'A torsiondrive dataset for benchmarking B3LYP-D3BJ/DEF2-QZVP ',
+  'store_wavefunction': 'none',
+  'implicit_solvent': None},
+ 'B3LYP-D3BJ/6-31+GSS ': {'method': 'B3LYP-D3BJ',
+  'basis': '6-31+gss',
+  'program': 'psi4',
+  'spec_name': 'B3LYP-D3BJ/6-31+GSS ',
+  'spec_description': 'A torsiondrive dataset for benchmarking B3LYP-D3BJ/6-31+GSS ',
+  'store_wavefunction': 'none',
+  'implicit_solvent': None},
+ 'B3LYP-D3BJ/6-311+GSS ': {'method': 'B3LYP-D3BJ',
+  'basis': '6-311+gss',
```

Comment:
To verify: we have listed here `GSS` and `gss`, but in the submittable `dataset.json.bz2`, we have `G**` and `g**`, respectively. Do we know which one QCEngine will take?

Reply:
From discussion: our submission includes `g**`, which should be what we want. The QCSubmit validation will convert these to `gss` for validation with basis-set-exchange, but push through `g**` for `psi4` execution.